install/assets/defaults/05-logging

```diff
@@ -19,7 +19,7 @@ FLUENTBIT_OUTPUT=${FLUENTBIT_OUTPUT:-"FORWARD"}
 FLUENTBIT_OUTPUT_FORWARD_HOST=${FLUENTBIT_OUTPUT_FORWARD_HOST:-"fluent-proxy"}
 FLUENTBIT_OUTPUT_FORWARD_TLS=${FLUENTBIT_OUTPUT_FORWARD_TLS:-"FALSE"}
 FLUENTBIT_OUTPUT_FORWARD_TLS_VERIFY=${FLUENTBIT_OUTPUT_FORWARD_TLS_VERIFY:-"FALSE"}
-FLUENTBIT_OUTPUT_LOKI_COMPRESS_GZIP=${FLUENTBIT_OUTPUT_LOKI_COMPRESS_GZIP:-"TRUE"}
+FLUENTBIT_OUTPUT_LOKI_COMPRESS_GZIP=${FLUENTBIT_OUTPUT_LOKI_COMPRESS_GZIP:-"gzip"}
```

Comment:
Just checking to see if this was meant to be there?

Reply:
Oh. Sorry.
This had to fix the fluent bit config. The correct config parameter must be `compress     gzip` but if I remember correctly the previous config had `compress     true`.
I see it's fixed already, so could you ignore this change then, please.